test/test_retryable_writes.py

```diff
@@ -454,6 +456,40 @@ def test_batch_splitting_retry_fails(self):
         self.assertEqual(coll.find_one(projection={'_id': True}), {'_id': 1})
 
 
+class TestWriteConcernError(IntegrationTest):
+    @client_context.require_version_min(3, 6)
```

Comment:
This needs 4.0 because the writeConcernError argument to failCommand was added in 4.0: https://jira.mongodb.org/browse/SERVER-35083

Reply:
Done.